res/css/views/right_panel/_ThreadPanel.pcss

```
@@ -20,11 +20,22 @@ limitations under the License.
 
     .mx_BaseCard_header {
         .mx_BaseCard_header_title {
+            .mx_BaseCard_header_title_heading {
+                margin-right: auto;
+            }
+
             .mx_AccessibleButton {
                 font-size: 12px;
                 color: $secondary-content;
             }
 
+            .mx_ThreadPanel_vertical_separator {
+                height: 16px;
+                margin-left: 12px;
```

```suggestion
                margin-left: var(--cpd-space-3x);
```

[compound spacing tokens](https://compound.element.io/?path=/docs/tokens-spacing--docs)